### PR TITLE
0.43 release

### PR DESCRIPTION
**Pull request by Cashmaney**

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -225,6 +230,13 @@ func NewSecretNetworkApp(
 	// add capability keeper and ScopeToModule for ibc module
 	app.capabilityKeeper = capabilitykeeper.NewKeeper(appCodec, keys[capabilitytypes.StoreKey], memKeys[capabilitytypes.MemStoreKey])
 
+	//	scopedIBCKeeper := app.CapabilityKeeper.ScopeToModule(ibchost.ModuleName)
+	//	scopedTransferKeeper := app.CapabilityKeeper.ScopeToModule(ibctransfertypes.ModuleName)
```
Comment (reuvenpo) [inline app/app.go:234]: what are these comments?

File: app/app.go [hunk 2]
```diff
@@ -459,14 +479,6 @@ func NewSecretNetworkApp(
...
 		}
-
-		// Initialize and seal the capability keeper so all persistent capabilities
-		// are loaded in-memory and prevent any further modules from creating scoped
-		// sub-keepers.
-		// This must be done during creation of baseapp rather than in InitChain so
-		// that in-memory capabilities get regenerated on app restart
-		ctx := app.BaseApp.NewUncachedContext(true, tmproto.Header{})
-		app.capabilityKeeper.InitializeAndSeal(ctx)
 	}
```
Comment (reuvenpo) [inline app/app.go:482]: why were these removed?

**Review by reuvenpo — COMMENTED** [review]
do we know what caused the tests that are being skipped there to fail?

File: app/app.go (hunk 1 above)
Comment (Cashmaney) [reply app/app.go:234]: Placeholders
